src/controller/controller.cpp

```diff
@@ -40,9 +40,7 @@ bool Controller::simulate() {
 
         auto visibles = world.getObjectsAround(individual->getPosition(), individual->getVision().length);
         auto action = individual->act(visibles);
-        std::cout << action->toString() << std::endl;
-        delete action;
```

Comment:
why remove the delete?

Reply:
I refactored the system to make the delete obsolete

Reply:
by passing the action by valur, not by reference, to be precise